examples/wordpress-mysql/bundle.json

```diff
@@ -0,0 +1,68 @@
+{
+    "name": "wordpress-mysql",
+    "version": "0.2.0",
+    "schemaVersion": "v1.0.0-WD",
```

Comment:
This should be `v1.0.0`

Reply:
Also, could you please update `duffle.json` from the `helloworld` example as well to the latest schema version?